admin/admin-domain-service/ballerina/resources/admin-api.yaml

```diff
@@ -2719,6 +2719,193 @@ paths:
+      parameters:
+        - $ref: '#/components/parameters/organizationId'
+      requestBody:
+        description: |
+          Organization object with updated information
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Organization'
+        required: true
+      responses:
+        200:
+          description: |
+            OK.
+            Label updated.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Organization'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        404:
+          $ref: '#/components/responses/NotFound'
+      security:
+        - OAuth2Security:
+            - apim:admin
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X PUT -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          -H "Content-Type: application/json" -d @data.json "https://127.0.0.1:9443/api/am/admin/organization/d7cf8523-9180-4255-84fa-6cb171c1f779"'
+      operationId: updateOrganization
+    delete:
+      tags:
+        - Organization (Individual)
+      summary: Delete an Organization
+      description: |
+        Delete an Organization by API Organization Id
+      parameters:
+        - $ref: '#/components/parameters/organizationId'
+      responses:
+        200:
+          description: |
+            OK.
+            Organization successfully deleted.
+          content: {}
+        404:
+          $ref: '#/components/responses/NotFound'
+      security:
+        - OAuth2Security:
+            - apim:admin
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X DELETE -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/admin/organization/d7cf8523-9180-4255-84fa-6cb171c1f779"'
+      operationId: removeOrganization
+  /organization-info/{claimValue}:
+    get:
+      tags:
+        - Organization (Individual)
+      summary: |
+        Authenticate Organization info
+      description: |
+        This operation can be used to authenticate Organization by specifying its claimValue.
+      parameters:
+        - $ref: '#/components/parameters/claimValue'
+      responses:
+        200:
+          description: |
+            OK.
+            Application details returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Organization'
+        400:
+          $ref: '#/components/responses/BadRequest'
+        404:
+          $ref: '#/components/responses/NotFound'
+      security:
+        - OAuth2Security:
+            - apim:admin
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -X GET -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/admin/organization/0a043c2b-ee75-4ef3-9e1c-fc2610ccfa8b"'
```

Comment:
curl example is invalid

Reply:
Fix https://github.com/wso2/apk/pull/630/commits/18c4a6d31a2cecda8c33bf21d01ff675e68bbf42